giraffe/src/components/Plot.tsx

```diff
@@ -51,10 +72,27 @@ export const Plot: FunctionComponent<Props> = ({
             </SizedTable>
           )
         }
+        if (config.staticLegend) {
+          resized = resizePlotWithStaticLegend(
```

Comment:
create a method that takes the resized variable to create the two different sizedPlotWith Legend in one place; DRY

Reply:
The width and heights are different though. I think DRY does not need to be applied fanatically even in cases where similar code is only repeated once. DRY is more for 3+ uses of something. We don't reach 3 here.

Reply:
i disagree, at the very least the two instances of it were confusing to me.

Reply:
The piece of code here is a simple `if` without an `else`.  And inside the `if` block is 2 statements.

^ That gets repeated once. It's two total usages. Each usage has different widths and heights. I don't think this is very confusing.

Reply:
My point is: DRY should not be fanatically applied. There should be some leeway with regards to not making everything into a function.

If the above piece of code were more complex, I would agree that DRY makes sense. Here, it's just unnecessary and just adds function bloat.